src/Monticello/MCTraitDefinition.class.st

```diff
@@ -58,22 +58,21 @@ MCTraitDefinition >> classSlotDefinitionString [
 
 { #category : #installing }
 MCTraitDefinition >> createClass [
+
 	| trait |
-	
 	trait := self class classInstaller make: [ :aBuilder |
-		aBuilder 
-			name: name;
-			traitComposition: (Smalltalk compiler evaluate: self traitCompositionString);
-			slots: self instanceVariables;
-			package:  category;
-			beTrait ].					
+		         aBuilder
+			         name: name;
+			         traitComposition: (Smalltalk compiler evaluate: self traitCompositionString);
+			         slots: self instanceVariables;
+			         package: self category;
+			         beTrait ].
 
 
 	trait ifNotNil: [
 		trait comment: comment stamp: commentStamp.
-		self classInstanceVariables ifNotEmpty: [
-			trait classSide slots: self classInstanceVariables ]].
-	^trait
+		self classInstanceVariables ifNotEmpty: [ trait classSide slots: self classInstanceVariables ] ].
```

Comment:
`MCClassDefinition >> #classInstanceVariables` looks a bit complex, so its result could be given to the block.
```suggestion
		self classInstanceVariables ifNotEmpty: [ :variables | trait classSide slots: variables ] ].
```